clear/main.py

```diff
@@ -449,6 +450,25 @@ def translations():
     cmd = "natclear -t"
     run_command(cmd)
 
+def remap_keys(dict):
+    return [{'key': k, 'value': v} for k, v in dict.items()]
+
+#
+# 'pbh' subcommand
+#
+@cli.command()
+@clicommon.pass_db
+def pbh(db):
```

Comment:
@vadymhlushko-mlnx please rework this part - use NAT pattern:
```bash
root@sonic:/home/admin# sonic-clear nat --help
Usage: sonic-clear nat [OPTIONS] COMMAND [ARGS]...

  Clear the nat info

Options:
  -h, -?, --help  Show this message and exit.

Commands:
  statistics    Clear all NAT statistics
  translations  Clear all NAT translations

root@sonic:/home/admin# show nat statistics

Protocol    Source    Destination    Packets    Bytes
----------  --------  -------------  ---------  -------
```

so eventually it will be something like this:
```bash
root@sonic:/home/admin# sonic-clear pbh statistics
```

Reply:
@vadymhlushko-mlnx please mark as resolved once it done